fix: add warning messages for redemptions

jpangelle opened the pull request:
### Description

This pull request adds warning messages for investors that are redeeming before linking their cent chain addresses.

The info blocks do not have the yellow background with the full-width style because I used the `Alert` component that is already used throughout the app. If we want to change the alert style, we should probably do it for all alerts to remain consistent.

Closes #254

onnovisser commented:
LGTM!

armandomistral reviewed (changes requested):
Thanks for the additions, @jpangelle! Yes, we should treat and apply the Alert component consistently. I tried making it consistent with the alerts used in the investment component (invest / redeem). So at some point we should merge both components into one, as they're both used inline. BTW I don't see a an alert added to the investment component in this PR. Is it missing?

jpangelle commented:
> Thanks for the additions, @jpangelle! Yes, we should treat and apply the Alert component consistently. I tried making it consistent with the alerts used in the investment component (invest / redeem). So at some point we should merge both components into one, as they're both used inline.

Hey Nico, I did a bit more digging. I was using the `Alert` component instead of the `Warning` component. I switched to using `Watching` in a follow-up commit. The messages should look like your mock ups now.

> BTW I don't see a an alert added to the investment component in this PR. Is it missing?

I am not sure what you mean. I added the three different warning messages specified in the Github Issue.

armandomistral commented:
Uh, thanks for the digging, @jpangelle. Good to know the distinction between `alert` and `warning`. The yellow warning  background looks good now, just the icon needs to be changed, according to the rule: alert -> help (i) icon, warning -> warning (!) icon. Also, let's add a bottom margin of 16px to the component the dropdown does not collide with the subsequent form control. 
![image](https://user-images.githubusercontent.com/23652850/126962921-4400c1ee-9b57-4ab1-84ca-5bd61bd31516.png)

jpangelle commented:
> Uh, thanks for the digging, @jpangelle. Good to know the distinction between `alert` and `warning`. The yellow warning background looks good now, just the icon needs to be changed, according to the rule:
> 
> * alert component -> help (i) icon
> * warning component -> warning (!) icon
> 
> Also, let's add a bottom margin of 16px to the component so the message does not collide with the subsequent form control (e.g. a dropdown).
> ![image](https://user-images.githubusercontent.com/23652850/126962921-4400c1ee-9b57-4ab1-84ca-5bd61bd31516.png)

Fixed!
